.dir-locals.el

```diff
@@ -0,0 +1,3 @@
+((nil . ((mocha-options . "--recursive --compilers js:@babel/register")
```

Comment:
speaking of the devil :) this is an emacs file right? i think the repository should be agnostic and this file should be gitignored